### DISCUSSION TITLE
Suggestion for fixing revs.

### DISCUSSION THREAD

**Discussion by ShootsTheMoon** [space-wizards/space-station-14#31846] [Ideaguys]
Here’s the idea:

The round begins with three players chosen as antagonists: two Head Revolutionaries and one Lead Revolutionary. These revolutionaries start with communication access, allowing them to coordinate and find a location to set up their forward operating base (FOB).

The Head Revolutionaries each start with 5 tokens, while the Lead Revolutionary starts with 5 tokens and a revolutionary flatpack. When the flatpack is deployed, it creates a stationary uplink that can be used to buy and sell items.
The revolutionaries will insert their tokens into the uplink and purchase special flashes, each costing 5 tokens, which can convert those flashed into revolutionaries (no more using regular flashes).
They then go out and flash up to 15 people, recruiting them to help steal high-value items like the digital board, CE mag boots, and PDAs from Security and Command (not the IDs).
Once these items are sold at the revolutionary uplink, they convert into more tokens that can be spent on additional special flashes, surplus rifles, ammo, revolutionary communications, pipe bombs, armor, makeshift laser pistols, and other items that encourage swarming tactics.

When the revolutionaries are discovered, or when they’ve exhausted their limited resources, the Lead Revolutionary (or a Head Revolutionary if the Lead is dead) can use the uplink to announce the uprising to the crew. This announcement, similar to War Ops, grants the revolutionaries additional tokens to spend on various items. However, it also causes all revolutionaries to receive an armband that identifies them to Security and the crew:

Regular revolutionaries get a red armband.
Head Revolutionaries get a blue armband.
The Lead Revolutionary gets a gold armband.

If the uplink is found and destroyed before the uprising announcement, or if the revolutionaries choose not to announce their presence, they won’t receive any identifying armbands, allowing them to continue operating in secret.

**Comment by N0tly** [comment]
I think the biggest problem is that any revolution comes down to dumb dm every time. Or at least on the servers I play on, it always does.
